util.py

```diff
@@ -498,7 +498,7 @@ def guess_rmantree_initial():
     rmantree = rmantree_from_env()
     if rmantree != '':
         vstr = rmantree.split('-')[-1]
-        vf = float(vstr.strip('/'))
```

Comment:
To be clear I think you have to strip / and \\ (one for windows and one for OSX/linux)  so strip('/\\')